test/functional/p2p-leaktests.py

```diff
@@ -150,6 +151,9 @@ def run_test(self):
         self.log.info("Service bits 5 and 7 are allowed after August 1st 2018")
         self.nodes[0].setmocktime(1533168000)  # August 2nd 2018
 
+        # change in mocktime causes bitcoind to disconnect P2P connections. Sleep for a second to allow that to complete before attempting to reconnect
+        time.sleep(1)
```

Comment:
Making things more robust by adding sleeps w/out checks is kind of meh. Can it check this precondition somehow and sleep longer if necessary?